content/en/docs/instrumentation/go/exporters.md

```diff
@@ -2,56 +2,178 @@
 	client := otlptracehttp.NewClient()
-	exporter, err := otlptrace.New(ctx, client)
-	if err != nil {
-		return nil, fmt.Errorf("creating OTLP trace exporter: %w", err)
-	}
-  	/* … */
+	return otlptrace.New(ctx, client)
 }
 ```
 
-To learn more on how to use the OTLP HTTP exporter, try out the
-[otel-collector](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector)
+### OTLP traces over gRPC
+
+[`go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc)
+contains an implementation of OTLP trace exporter using gRPC.
+
+Here's how you can create an exporter with default configuration:
+
+```go
+import (
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+func newExporter(ctx context.Context) (trace.SpanExporter, error) {
+	client := otlptracegrpc.NewClient()
+	return otlptrace.New(ctx, client)
+}
+```
 
 ### Jaeger
 
 To try out the OTLP exporter, since v1.35.0 you can run
 [Jaeger](https://www.jaegertracing.io/) as an OTLP endpoint and for trace
-visualization in a docker container:
+visualization in a Docker container:
 
 ```shell
 docker run -d --name jaeger \
   -e COLLECTOR_OTLP_ENABLED=true \
   -p 16686:16686 \
+  -p 4317:4317 \
   -p 4318:4318 \
   jaegertracing/all-in-one:latest
 ```
 
-## Prometheus
+### OTLP metrics over HTTP (Experimental)
+
+[`go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp)
+contains an implementation of OTLP metrics exporter using HTTP with binary
+protobuf payloads.
+
+Here's how you can create an exporter with default configuration:
+
+```go
+import (
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+func newExporter(ctx context.Context) (metric.Exporter, error) {
+	return otlpmetrichttp.New(ctx)
+}
+```
+
+### OTLP metrics over gRPC (Experimental)
+
+[`go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc)
+contains an implementation of OTLP trace exporter using gRPC.
+
+Here's how you can create an exporter with default configuration:
+
+```go
+import (
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+func newExporter(ctx context.Context) (metric.Exporter, error) {
+	return otlpmetricgrpc.New(ctx)
+}
+```
+
+## Prometheus (Experimental)
+
+[`go.opentelemetry.io/otel/exporters/prometheus`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/prometheus)
+contains an implementation of Prometheus metrics exporter.
+
+Here's how you can create an exporter (which is also a metric reader) with
```

Comment:
well... It isn't technically a metric.Exporter.  I wonder if there is a better way to avoid confusing users here.

Ideas:

* Make all functions "newReader(ctx) (metric.Reader, error).  Push exporters return a newPeriodicReader with the appropriate exporter.
* Change this example to say "Here's how you can create a reader with default configuration:"
* Leave it as-is

Reply:
I like this idea:

> Make all functions "newReader(ctx) (metric.Reader, error)

Reply:
I think it's probably fine to leave as is. I expect most end-users won't differentiate too much between the technical definition of a metric exporter and the looser definition of an "exporter". If it's a way to emit data to prometheus then that aligns with the latter definition.